_includes/favicon.html

```diff
@@ -0,0 +1,23 @@
+{%- comment -%}
```

Comment:
Mostly a note to myself: we should tell users that we're adding a new `_includes` file; if they've already created one, they'll need to migrate.

Reply:
Good point: the file gets included in the `head` of each page, so it should be a `link` element.

Moreover, as it's cached, it should be `page`-independent. Are favicons always site-wide, or is it possible to have different favicons on different pages of the same site?

Reply:
> Are favicons always site-wide, or is it possible to have different favicons on different pages of the same site?

It's certainly possible to have different favicons on different pages of the same site, though that does seem a bit uncommon. In general, this would require them to already "bail out" of some of our theme (since they'd need to override `head_Custom` on a per-page basis, or use a `page.`-scoped variable to set the favicon).

We can also include migration instructions that tell users how to circumvent the page cache. The relevant paragraph from [the plugin README](https://github.com/benbalter/jekyll-include-cache/blob/main/README.md#one-potential-gotcha) is:

> For Jekyll Include Cache to work, you cannot rely on the page context to pass variables to your include (e.g., `assign foo=bar` or `page.title`). Instead, you must explicitly pass all variables to the include as arguments, and reference them within the include as `include.foo` (instead of `page.foo` or just `foo`).

Reply:
An alternative might be to add a config option for whether to cache the favicon-include or not. If it's not cached, it can refer to `page`.

But we could defer that until somebody requests it.

Reply:
If this will be a `0.6` change, I think we can defer it until somebody requests it.